lib/ex_unit/lib/ex_unit/formatter.ex

```diff
@@ -118,6 +118,30 @@ defmodule ExUnit.Formatter do
     <> report(tags, failures, width, formatter)
   end
 
+  @doc false
+  def format_assertion_error(%ExUnit.AssertionError{} = struct, width, formatter, counter_padding \\ @counter_padding) do
+    padding_size = byte_size(@inspect_padding)
+
+    fields = [
+      note: if_value(struct.message, &format_banner(&1, formatter)),
+      code: if_value(struct.expr, &code_multiline(&1, padding_size)),
+      lhs:  if_value(struct.left,  &inspect_multiline(&1, padding_size, width)),
+      rhs:  if_value(struct.right, &inspect_multiline(&1, padding_size, width))
+    ]
+
+    fields =
+      if formatter.(:colors_enabled?, nil) do
```

Comment:
Let's change `:colors_enabled?` to `:diff_enabled?` and make the default be `false`. This way we won't need to check for it here: https://github.com/elixir-lang/elixir/pull/4709/files#diff-699b0020c6a196231a7d473322f1744dR24

We probably use colors_enabled? in a couple other places, so let's not forget to change those. :)